runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -182,6 +186,29 @@ public void onGameTick(GameTick gameTick)
 		checkActionTimeout();
 	}
 
+	@Subscribe
+	public void onVarbitChanged(VarbitChanged varbitChanged)
+	{
+		if (config.notifyWhenRoundStarting() != TimerAlertOptions.NONE)
```

Comment:
Generally the way we do varbit changes is to store the old value and then detect a change, and then check things like config etc, since it is faster. You can also use it to detect the duplicate alerts.

Reply:
Done